inc/ui-controller.php

```diff
@@ -5,6 +5,8 @@
  * @package CSV_Import_Framework
  */
 
+//phpcs:disable WordPress.Security.NonceVerification.NoNonceVerification,WordPressVIPMinimum.Variables.VariableAnalysis.UnusedVariable,WordPressVIPMinimum.Files.IncludingFile.UsingCustomConstant
```

Comment:
You may want to disable nonce verification on only the specific lines where it is used versus the entire file.